machine_manager_preventive/wizard/create_repair_order.py

```diff
@@ -132,5 +136,5 @@ def create_repair_from_pmo(self):
             value['view_mode'] = 'form,tree'
         else:
             value['domain'] = (u'{}{}{}'.format(
-                "[('id','in',[", ','.join(map(str, [repair.id])), "])]"))
+                "[('id','in',[", ','.join(map(str, repairs)), "])]"))
```

Comment:
Y finalmente aquí pondría value['domain'] = [('id', 'in', repairs.ids]
Yo lo hubiera hecho así.